rocAL/rocAL/source/meta_data/coco_meta_data_reader.cpp

```diff
@@ -197,10 +382,12 @@ void COCOMetaDataReader::read_all(const std::string &path)
                     if (0 == std::strcmp(internal_key, "width"))
                     {
                         img_size.w = parser.GetInt();
+                        max_width = img_size.w > max_width? img_size.w : max_width;
                     }
                     else if (0 == std::strcmp(internal_key, "height"))
                     {
                         img_size.h = parser.GetInt();
+                         max_height = img_size.h > max_height? img_size.h : max_height;
```

Comment:
max_w/max_h is always set as 0 so unless there's a set function for the values, is this comparison needed?

Reply:
We can discuss on this